main/src/ca/uwaterloo/flix/api/lsp/provider/FindReferencesProvider.scala

```diff
@@ -80,9 +82,10 @@ object FindReferencesProvider {
     ("status" -> "success") ~ ("result" -> locs.map(_.toJSON))
   }
 
-  private def findDefUses(sym: Symbol.DefnSym)(implicit index: Index, root: Root): JObject = {
+  private def findDefReferences(sym: Symbol.DefnSym)(implicit index: Index, root: Root): JObject = {
+    val defn = Location.fromDefnSym(sym)
```

Comment:
I think we can just do `Location.from(sym.loc)`. We dont need all the extra functions :)

Reply:
Yes, perfect!